### ISSUE TITLE
Simpleshare uploads failing

### ISSUE THREAD

**Issue by aarondunn** [aarondunn/simpleshare#79]
report from user

"Hi, simple share is a great tool and the only why I use box at all. Lately I get more and more failed uploads. Any idea how to fix that?

I am on OS 10.10.3. It happens with various file sizes. I am usually uploading zip files. "

**Comment by aarondunn** [comment]
@Rognarroc definitely still an issue, at least for the version on the app store. I can test a newer one if you want. I am using the latest osx

**Comment by Rognarroc** [comment]
In issue #70 you wrote you are not seeing uploads falis lately...

**Comment by aarondunn** [comment]
Yes not general ones, but for zip files i see it in the version in the app store. If you want to send me a newer build i can re-test
